cms/tests/frontend/integration/pagetree.js

```diff
@@ -13,55 +13,55 @@ var xPath = casperjs.selectXPath;
 var createJSTreeXPathFromTree = cms.createJSTreeXPathFromTree;
 var getPasteHelpersXPath = cms.getPasteHelpersXPath;
 
-var closeWizard = function() {
-    return function() {
+var closeWizard = function () {
```

Comment:
Formatting in this file should not have occurred.

Reply:
Automatic linting mistake. I undid this.